### PR TITLE
fix(starfish): align more table columns and span info

### PR DESCRIPTION
**Pull request by DominikB2014**
2 changes.
1. (endpoint overview) Right align the duration column in the sample events page
image

2. Left align the throughput and duration in the sample list sidebar. I'm not sure if i'm the biggest of doing this via `containerProps` if there's any other quick suggestions lmk!

image

### CODE REVIEW

File: static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx [hunk 1]
```diff
@@ -28,13 +30,23 @@ function SampleInfo(props: Props) {
     'span-summary-panel-metrics'
   );
 
+  const style: CSSProperties = {
+    textAlign: 'left',
```
Comment (gggritso) [inline static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx:34]: Left? Not right?

Comment (DominikB2014) [reply static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx:34]: Yeah, that way the number is actually under the `throughput` and `Duration (p95)` unlike rn
image

Comment (gggritso) [reply static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx:34]: Oh my God, cells are used outside of tables, right, I forgot. Oof.

**Review by gggritso — APPROVED** [review]

File: static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx:34]: Yup lol, It was done that way to keep all formatting consistent `(/s)`, can definitely do something else going forward.

**Pull request merged**